src/blog/2024/04/beginers-guide-to-node-red-varriables.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Beginner's Guide to Node-RED Variables
+subtitle: Understanding Flow, Global, and Environment Variables in Node-RED.
```

Comment:
What do you think about having the subtitle as the title and vice versa? I feel it's a little more direct on what the blog is all about and having these keywords in the title can help with SEO better.

Reply:
I agree, let's make that change.